src/Umbraco.Web.UI.Client/src/packages/media/dropzone/components/input-dropzone/input-dropzone.element.ts

```diff
@@ -0,0 +1,300 @@
+import { UmbDropzoneChangeEvent, UmbDropzoneManager, UmbDropzoneSubmittedEvent } from '../../index.js';
+import type { UmbUploadableItem } from '../../types.js';
+import { UmbFileDropzoneItemStatus } from '../../constants.js';
+import {
+	css,
+	customElement,
+	html,
+	ifDefined,
+	nothing,
+	property,
+	query,
+	repeat,
+	state,
+	when,
+} from '@umbraco-cms/backoffice/external/lit';
+import type { UUIFileDropzoneElement, UUIFileDropzoneEvent } from '@umbraco-cms/backoffice/external/uui';
+import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
+import { UmbTextStyles } from '@umbraco-cms/backoffice/style';
+import { formatBytes } from '@umbraco-cms/backoffice/utils';
+import { UmbFormControlMixin } from '@umbraco-cms/backoffice/validation';
+
+/**
+ * @element umb-input-dropzone
+ * @fires ProgressEvent When the progress of the upload changes.
+ * @fires UmbDropzoneChangeEvent When the upload is complete.
+ * @fires UmbDropzoneSubmittedEvent When the upload is submitted.
+ * @slot - The default slot.
+ */
+@customElement('umb-input-dropzone')
+export class UmbInputDropzoneElement extends UmbFormControlMixin<UmbUploadableItem[], typeof UmbLitElement>(
```

Comment:
From a shift look it seems like this component does not have any specific bounds to the Media package. So maybe it as well could be in the dropzone package? :-)

Reply:
Yes, you are right. It is more agnostic than media although it can also create a media item. It is already physically placed in the dropzone package, but as you discovered it is re-exported from the media package itself along with the other stuff so as not to cause a breaking change.

One could argue that the dropzone package itself should not be placed in **packages/media** but instead have its very own **package/dropzone** folder. What do you think?